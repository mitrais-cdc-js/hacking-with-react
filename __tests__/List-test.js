jest.mock('react-dom');

import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import List from '../src/pages/List';

describe('List', () => {
    it('renders three repo links', () => {
        const rendered = renderer.create(
            <List />
        );

        expect(rendered.toJSON()).toMatchSnapshot();
    });
});