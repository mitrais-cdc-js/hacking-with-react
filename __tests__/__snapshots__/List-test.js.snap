exports[`List renders three repo links 1`] = `
<div>
  <p>
    You are here: 
    <a
      onClick={[Function]}
      style={Object {}}>
      Home
    </a>
  </p>
  <p>
    Please choose a repository from the list below.
  </p>
  <ul>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}>
        React
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}>
        React Native
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}>
        Jest
      </a>
    </li>
  </ul>
</div>
`;
